external/coreclr/project.json.template

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "frameworks": {
+    "{TFM}": {
+      "dependencies": {
+        "Microsoft.NETCore.Platforms": "1.2.0-beta-24721-02",
+        "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24728-02"
```

Comment:
Should we add the TestHost in here as well? or do you want us to have it on the Xunit depproj instead?